docs/build-with-powerloom/devnet/FAQs.md

```diff
@@ -0,0 +1,95 @@
+    
+### What types of OS can I use to run the Snapshotter node?
+You can use the macOS, Linux, or Windows to run the Snapshotter node. For more details on setting up and running the node, visit [here](https://docs.powerloom.io/docs/build-with-powerloom/devnet/getting-started).
+    
+### What is a burner wallet and how do you generate one? 
+A burner/disposable wallet is a type of cryptocurrency wallet that is designed for temporary use. Burner wallets are often used for small transactions or events where security and long-term storage are not priorities.
+
+**Note:** Please do not use your NFT minting wallet to run the Snapshotter lite node. The safest approach is to generate and assign a burner wallet using tools like [Vanity-ETH](https://vanity-eth.tk/) and [Powerloom Burner Wallet Generator](https://snapshotter-dashboard.powerloom.network/burner)
+
+### What security measures should I consider when creating a burner wallet?
+When creating a burner wallet, prioritize security by managing private keys securely, limiting funds stored, performing offline transactions, regularly backing up data, ensuring a secure environment, using trusted sources, considering multi-signature wallets, and staying informed about security practices.
+    
+### How do I deploy my Snapshotter node once the environment is configured?
+After configuring the environment variables, set the powerloom_deploy codebase by running the `bootstrap.sh` command in the terminal. Developers may use different shells, such as bash or zsh.
+
+**Note**: This is a one-time step that resets the `powerloom_deploy` codebase to the latest version of the `devnet` branch. After completing this one-time setup, you'll be ready to dive into the codebase and start building amazing data applications!
+    
+### What chains are supported?
+We support Ethereum Mainnet and Sepolia Testnet.
+    
+### What steps should I follow to perform a clean reset of my node?
+You can do a clean **reset** by running the following command before restarting.
+```markdown
+    docker-compose --profile ipfs down --volumes
+```
+or
+```markdown
+    docker compose --profile ipfs down --volumes
+```
+    
+### How can I stop the node?
+To shut down services, use the keyboard shortcut **`Ctrl + C`** on macOS/Linux or **`Ctrl + Break`** on Windows. Run the provided Docker command before restarting the node setup for a clean reset.
+    
+### Can we run the setup on different languages?
+Yes, you can run the setup in different languages. The Snapshotter node is a complex system with multiple components written in different languages. Pooler is a Python program, while Audit Protocol is coded in Go. Since Pooler and Audit Protocol communicate via RabbitMQ, either component can be ported to different languages, making the Snapshotter node accessible to developers unfamiliar with Python or Go.
+    
+### What is the purpose of the PROTOCOL_STATE_CONTRACT variable?
```

Comment:
this should clarify that there are different protocol state contracts depending on the data source contract's chain as well as the epoch size on that chain that is preferred by devs. An epoch size of 1 means more real time use cases, 10 would mean better suited for aggregated dataset use cases like data dashboards

Reply:
The PROTOCOL_STATE_CONTRACT variable serves a crucial role by specifying the address of the protocol state contract. Its configuration depends on a couple of key factors:

Data Source Contract's Chain: The specific blockchain on which the data source contract is deployed influences the protocol state contract address.

Epoch Size Preference: The epoch size, chosen based on the application’s needs, affects how data is processed and utilized:

- Epoch Size of 1: Ideal for real-time use cases where immediate data processing is crucial.
- Epoch Size of 10: Better suited for use cases that involve aggregated data sets, like data dashboards, where data can be compiled and reviewed over a set period.

This setup allows developers to adapt the protocol according to varying requirements and use cases, ensuring flexibility across different blockchain platforms.